tests/core/gui/test_invite_user.py

```diff
@@ -0,0 +1,66 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import pytest
+from PyQt5 import QtCore
+
+
+from parsec.core.gui.lang import translate as _
+
+from tests.common import customize_fixtures
+
+
+@pytest.mark.gui
+@pytest.mark.trio
+@customize_fixtures(logged_gui_as_admin=True)
+async def test_cancel_user_invitation(
+    qapp, aqtbot, logged_gui, running_backend, backend, autoclose_dialog, monkeypatch, alice
+):
+
+    email = "i@like.coffee"
+
+    # Patch dialogs
+    monkeypatch.setattr("parsec.core.gui.users_widget.get_text_input", lambda *x, **y: email)
+    monkeypatch.setattr(
+        "parsec.core.gui.users_widget.ask_question",
+        lambda *x, **y: _("TEXT_USER_INVITE_CANCEL_INVITE_ACCEPT"),
+    )
+    u_w = await logged_gui.test_switch_to_users_widget()
+
+    # Force send_email to false
+    _invite_new = u_w.core._backend_conn.cmds.invite_new
```

Comment:
no need for this you can use the decorator `@customize_fixtures(backend_has_email=True)`, then the fixture `email_letterbox` (see `tests/core/gui/users_widgets/test_invite.py`)

Reply:
it's much better :)